contrib/passwd.bzl

```diff
@@ -54,17 +41,32 @@ passwd_file = rule(
         "info": attr.string(default = "user"),
         "home": attr.string(default = "/home"),
         "shell": attr.string(default = "/bin/bash"),
-        "build_tar": attr.label(
-            default = Label("@bazel_tools//tools/build_defs/pkg:build_tar"),
-            cfg = "host",
-            executable = True,
-            allow_files = True,
-        ),
+        "output_file": attr.string(default = ""),
     },
     executable = False,
     outputs = {
-        "out": "%{name}.passwd",
-        "tar": "%{name}.passwd.tar",
+        "out": "%{output_file}" or "%{name}",
     },
     implementation = _impl,
 )
+
+def passwd_tar(name, username, uid, gid, info, home, shell, passwd_file_name=""):
```

Comment:
Do we have a use case where the tarball contains anything other than `/etc/passwd`?

Reply:
not that i know of. I can always default it to "/etc/passwd"